strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/config/StorageApiConfig.java

```diff
@@ -33,16 +32,6 @@
     @Inject
     private List<ArtifactCoordinatesValidator> versionValidators;
 
-    @Bean(name = "checksumCacheManager")
-    ChecksumCacheManager checksumCacheManager()
```

Comment:
Has this been moved elsewhere, or are these settings the defaults now?

Reply:
`ChecksumCacheManager` is marked as `@Component` now

Reply:
I see, thanks! :)